Prod deployment failing

A flyway error is preventing the current version in `main` from starting up successfully

```
Jan 31 15:22:24  	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:309) ~[spring-beans-7.0.3.jar!/:7.0.3]
Jan 31 15:22:24  	... 15 common frames omitted
Jan 31 15:22:24  Caused by: org.flywaydb.core.api.exception.FlywayValidateException: Validate failed: Migrations have failed validation
Jan 31 15:22:24  Detected resolved migration not applied to database: 9.
Jan 31 15:22:24  To ignore this migration, set -ignoreMigrationPatterns='*:ignored'. To allow executing this migration, set -outOfOrder=true.
Jan 31 15:22:24  Need more flexibility with validation rules? Learn more: https://rd.gt/3AbJUZE
Jan 31 15:22:24  	at org.flywaydb.core.Flyway.lambda$migrate$3(Flyway.java:199) ~[flyway-core-11.14.1.jar!/:na]
Jan 31 15:22:24  	at org.flywaydb.core.FlywayExecutor.execute(FlywayExecutor.java:240) ~[flyway-core-11.14.1.jar!/:na]
Jan 31 15:22:24  	at org.flywaydb.core.Flyway.migrate(Flyway.java:186) ~[flyway-core-11.14.1.jar!/:na]
Jan 31 15:22:24  	at org.springframework.boot.flyway.autoconfigure.FlywayMigrationInitializer.afterPropertiesSet(FlywayMigrationInitializer.java:67) ~[spring-boot-flyway-4.0.2.jar!/:4.0.2]
Jan 31 15:22:24  	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.invokeInitMethods(AbstractAutowireCapableBeanFactory.java:1864) ~[spring-beans-7.0.3.jar!/:7.0.3]
Jan 31 15:22:24  	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1813) ~[spring-beans-7.0.3.jar!/:7.0.3]
Jan 31 15:22:24  	... 22 common frames omitted
Jan 31 15:22:24  
Jan 31 15:22:24  ERROR component terminated with non-zero exit code: 1,
```

--- a/backend/src/main/resources/application.yml
+++ b/backend/src/main/resources/application.yml
@@ -21,6 +21,7 @@ spring:
     enabled: true
     locations: classpath:db/migration
     baseline-on-migrate: true
+    out-of-order: true
 
   cache:
     type: caffeine

--- a/backend/src/main/resources/db/migration/V9__game_scheduled_time.sql
+++ b/backend/src/main/resources/db/migration/V9__game_scheduled_time.sql
@@ -1,5 +1,5 @@
 -- Add scheduled game time for calendar views
-ALTER TABLE games ADD COLUMN scheduled_time TIME;
+ALTER TABLE games ADD COLUMN IF NOT EXISTS scheduled_time TIME;
 
 -- Create index for time-based ordering within a date
-CREATE INDEX idx_games_date_time ON games(game_date, scheduled_time);
+CREATE INDEX IF NOT EXISTS idx_games_date_time ON games(game_date, scheduled_time);

